### PR TITLE
Update doc.go

### PR DESCRIPTION
**Pull request by himanigulati01**
Fixed the code documentation.

### CODE REVIEW

**Comment by mikebz** [comment]
@karlkfi I think this is waiting for approval as well.

**Comment by mikebz** [comment]
@janetkuo do you mind hitting approve here if this looks good

**Review by janetkuo — APPROVED** [review]

**Pull request merged**
